src/style/tokens/layout.css

```
@@ -14,7 +14,11 @@
 
   /* Viewport Widths — modal & layout patterns */
   --vw-100: 100vw;
+  --vw-100: 100vw;
```

Stylelint (17.4.0)

[error] Unexpected duplicate "--vw-100" (declaration-block-no-duplicate-custom-properties)

(declaration-block-no-duplicate-custom-properties)